modules/integration/src/test/scala/scala/cli/integration/SipScalaTests.scala

```diff
@@ -607,15 +607,19 @@ class SipScalaTests extends ScalaCliSuite with SbtTestHelper with MillTestHelper
         .fromRoot { root =>
           val localRepoPath = root / "local-repo"
           val sv            = "3.4.1-RC1"
-          val csRes = os.proc(
-            TestUtil.cs,
-            "fetch",
-            "--cache",
-            localRepoPath,
-            s"org.scala-lang:scala3-compiler_3:$sv"
-          )
-            .call(cwd = root)
-          expect(csRes.exitCode == 0)
+          val artifactNames =
+            Seq("scala3-compiler_3") ++ (if (withBloop) Seq("scala3-sbt-bridge") else Nil)
```

Comment:
Theoretically there is also an artifact for newer scala 2 versions `scala2-sbt-bridge` -> but we can do a follow up later on

Reply:
https://github.com/VirtusLab/scala-cli/issues/2874
I created an issue.